frontend/src/views/coverage/enrollment/AddVisaResidentWithPHN.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div v-if="pageAction === this.PAGE_ACTION.PHN_SEARCH">
+    <ResidentPHN @update-resident="updateResident" />
+  </div>
+  <div v-else-if="pageAction === this.PAGE_ACTION.STUDENT_REGISTRATION">
+    <ResidentDetails :resident="this.getPersonDetailsResult?.person" @register-resident="registerResident" />
+  </div>
+  <div v-else-if="pageAction === this.PAGE_ACTION.CONFIRMATION">
+    <AppRow>
```

Comment:
It might be cleaner if this is split into it's own component. Trivial yes but but it would make the display logic cleaner. Your call.

Reply:
Had considered adding one but thought it might be overkill, added now and it looks a lot better.